packages/aws-rfdk/lib/deadline/test/worker-fleet.test.ts

```diff
@@ -92,12 +94,18 @@ beforeEach(() => {
 
 test('default worker fleet is created correctly', () => {
   // WHEN
+  class userDataProvider extends InstanceUserDataProvider{
+    preCloudWatchAgent(host: IHost) {
+      host.userData.addCommands('mkdir test_dir');
+    }
+  }
   const fleet = new WorkerInstanceFleet(wfstack, 'workerFleet', {
     vpc,
     workerMachineImage: new GenericWindowsImage({
       'us-east-1': 'ami-any',
     }),
     renderQueue,
+    userDataProvider: new userDataProvider(),
```

Comment:
I'm not sure we need this to be added here. The test appears to be specifically for testing defaults and this new user data provider is optional.

Reply:
Sure. It just artifacts.